src/Image.tsx

```diff
@@ -52,6 +53,7 @@ const ImageInternal: CompoundedComponent<ImageProps> = ({
   height,
   style,
   preview = true,
+  previewSrc,
```

Comment:
This prop better put into `preview: { src }`

Reply:
👌ok